xbmc/addons/AddonVersion.h

```diff
@@ -1,3 +1,4 @@
+#pragma once
```

Comment:
FYI, unlike #define guards, `#pragma once` can appear anywhere in the file with the same effect, so I usually like placing the copyright header first